roles/openshift_health_checker/openshift_checks/docker_image_availability.py

```diff
@@ -65,16 +69,14 @@ def run(self, tmp, task_vars):
 
     def required_images(self, task_vars):
         deployment_type = get_var(task_vars, "deployment_type")
-        # FIXME(juanvallejo): we should handle gracefully with a proper error
-        # message when given an unexpected value for `deployment_type`.
-        image_base_name = self.docker_image_base[deployment_type]
+        if deployment_type not in self.docker_image_base:
+            msg = "Unable to determine required images for deployment type {deployment}"
+            raise OpenShiftCheckException(msg.format(deployment=deployment_type))
+
+        image_base_name = self.docker_image_base.get(deployment_type)
 
         openshift_release = get_var(task_vars, "openshift_release")
-        # FIXME(juanvallejo): this variable is not required when the
-        # installation is non-containerized. The example inventories have it
-        # commented out. We should handle gracefully and with a proper error
-        # message when this variable is required and not set.
-        openshift_image_tag = get_var(task_vars, "openshift_image_tag")
+        openshift_image_tag = get_var(task_vars, "openshift_image_tag", default=openshift_release[1:])
```

Comment:
Why are we slicing out the first character? It may be a 'v', or it may not.... @sosiouxme has done some work with normalizing the release version.